src/spyglass/spikesorting/v1/sorting.py

```diff
@@ -0,0 +1,318 @@
+
+import os
+import tempfile
+import time
+
+import datajoint as dj
+import numpy as np
+import pynwb
+import spikeinterface as si
+import spikeinterface.extractors as se
+import spikeinterface.preprocessing as sip
+import spikeinterface.sorters as sis
+from spikeinterface.sortingcomponents.peak_detection import detect_peaks
+
+from spyglass.common.common_lab import LabMember, LabTeam
+from spyglass.common.common_interval import IntervalList
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+from spyglass.utils.dj_helper_fn import fetch_nwb
+from spyglass.spikesorting.v1.recording import (
+    SpikeSortingRecording,
+    SpikeSortingRecordingSelection,
+)
+from .recording import _consolidate_intervals
+
+schema = dj.schema("spikesorting_v1_sorting")
+
+
+@schema
+class SpikeSorterParameter(dj.Lookup):
+    definition = """
+    sorter: varchar(200)
+    sorter_param_name: varchar(200)
+    ---
+    sorter_param: blob
+    """
+    contents = [
+        [
+            "mountainsort4",
+            "franklab_tetrode_hippocampus_30KHz",
+            {
+                "detect_sign": -1,
+                "adjacency_radius": 100,
+                "freq_min": 600,
+                "freq_max": 6000,
+                "filter": False,
+                "whiten": True,
+                "num_workers": 1,
+                "clip_size": 40,
+                "detect_threshold": 3,
+                "detect_interval": 10,
+            },
+        ],
+        [
+            "mountainsort4",
+            "franklab_probe_ctx_30KHz",
+            {
+                "detect_sign": -1,
+                "adjacency_radius": 100,
+                "freq_min": 300,
+                "freq_max": 6000,
+                "filter": False,
+                "whiten": True,
+                "num_workers": 1,
+                "clip_size": 40,
+                "detect_threshold": 3,
+                "detect_interval": 10,
+            },
+        ],
+        [
+            "clusterless_thresholder",
+            "default_clusterless",
+            {
+                "detect_threshold": 100.0,  # uV
+                # Locally exclusive means one unit per spike detected
+                "method": "locally_exclusive",
+                "peak_sign": "neg",
+                "exclude_sweep_ms": 0.1,
+                "local_radius_um": 100,
+                # noise levels needs to be 1.0 so the units are in uV and not MAD
+                "noise_levels": np.asarray([1.0]),
+                "random_chunk_kwargs": {},
+                # output needs to be set to sorting for the rest of the pipeline
+                "outputs": "sorting",
+            },
+        ],
+    ]
+    sorters = sis.available_sorters()
+    contents.extend(
+        [
+            [sorter, "default", sis.get_default_sorter_params(sorter)]
+            for sorter in sorters
+        ]
+    )
+
+    @classmethod
+    def insert_default(cls):
+        cls.insert1(cls.contents, skip_duplicates=True)
```

Comment:
I don't think this will ever actually insert. Lookup gets the contents as soon as it's declared. If we want to keep based on assumptions people will have from other similarly structured tables, I would...
```suggestion
        pass
```

Reply:
I kept this in case someone deletes the row from `contents` and it needs to be re-inserted. Is this not possible?